### DISCUSSION TITLE
Trend Panel Plugin feature request (zoom in and out)

### DISCUSSION THREAD

**Discussion by menukagurung** [grafana/grafana#69992] [Dashboards, Panels & Visualizations]
Do we have any plans for the following?

- Zoom to re-query the data somehow
- A visual zoom for easier hovering/interaction

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82666. Feel free to continue the discussion around this there. Thank you!
